backend/prisma/schema.prisma

```diff
@@ -0,0 +1,165 @@
+    channelLog           ChannelLog[]
+}
+
+model UserProfile {
+    id       Int    @id @default(autoincrement())
+    user_id  Int
+    username String @unique
+    bio      String
+    location String
+    avatar   String
+    level    Float
+    User     User   @relation(fields: [user_id], references: [id])
+}
+
+model CareerLog {
+    id          Int             @id @default(autoincrement())
+    user_id     Int
+    opponent_id Int
+    result      CareerLogResult
+    timestamp   DateTime        @default(now())
+    User        User            @relation(name: "UserLog", fields: [user_id], references: [id])
+    Opponent    User            @relation(name: "OpponentLog", fields: [opponent_id], references: [id])
+}
+
+model Achievement {
+    id             Int              @id @default(autoincrement())
+    name           String           @unique
+    avatar         String
+    description    String
+    condition      String           @unique
+    type           String
+    AchievementLog AchievementLog[]
+}
+
+model AchievementLog {
+    id             Int         @id @default(autoincrement())
+    user_id        Int
+    achievement_id Int
+    score          Int
+    timestamp      DateTime    @default(now())
+    User           User        @relation(fields: [user_id], references: [id])
+    Achievement    Achievement @relation(fields: [achievement_id], references: [id])
+}
+
+model Notification {
+    id              Int               @id @default(autoincrement())
+    title           String
+    body            String
+    type            String
+    sender_id       Int
+    recipient_id    Int
+    timestamp       DateTime          @default(now())
+    User            User              @relation(name: "SentNotification", fields: [sender_id], references: [id])
+    Recipient       User              @relation(name: "ReceivedNotification", fields: [recipient_id], references: [id])
+    NotificationLog NotificationLog[]
+}
+
+model NotificationLog {
+    id              Int          @id @default(autoincrement())
+    user_id         Int
+    notification_id Int
+    readAt          DateTime?
+    User            User         @relation(fields: [user_id], references: [id])
+    Notification    Notification @relation(fields: [notification_id], references: [id])
+}
+
+model Friendship {
+    id        Int              @id @default(autoincrement())
+    user_id   Int
+    friend_id Int
+    status    FriendshipStatus
+    createdAt DateTime         @default(now())
+    updatedAt DateTime         @updatedAt
+    User      User             @relation(name: "SentFriendship", fields: [user_id], references: [id])
+    Friend    User             @relation(name: "ReceivedFriendship", fields: [friend_id], references: [id])
+}
+
+model Chat {
+    id           Int       @id @default(autoincrement())
+    sender_id    Int
+    recipient_id Int
+    message      String
+    sentAt       DateTime  @default(now())
+    ReadAt       DateTime?
```

Comment:
Verify case in all the models
(ReadAt should be in lowercase)

Reply:
done ✅